src/Providers/ControllersServiceProvider.php

```diff
@@ -19,4 +24,10 @@ public function register()
         $this->app->register(MacroServiceProvider::class);
         $this->app->register(RevisionsServiceProvider::class);
     }
+
+    protected function updateDefaultLaravelResolvers()
```

Comment:
missed description in added function. Also IMHO it not **updates** but **replaces**